sympy/simplify/radsimp.py

```diff
@@ -527,6 +527,28 @@ def collect_const(expr, *vars, **kwargs):
     targeted. Although any Number can also be targeted, if this is not
     desired set ``Numbers=False`` and no Float or Rational will be collected.
 
+    Parameters
+    ==========
+
+    expr : Numbers,Symbols
+           This parameter defines the expression to be worked upon by the function.
```

Comment:
the expression from which terms with similar coefficient are to be collected. A non-Add expression is returned as it is.